examples/access-plugin-minimal/watcherjob.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+)
+
+func (g *googleSheetsClient) HandleEvent(ctx context.Context, event types.Event) error {
+	if event.Resource == nil {
+		return nil
+	}
+
+	r := event.Resource.(types.AccessRequest)
+
+	if r.GetState() == types.RequestState_PENDING {
+		if err := g.createRow(r); err != nil {
+			return err
+		} else {
```

Comment:
You can eliminate the `else` here. The fact that the if case ends with a return means everything after that is a de-facto else.